metabase/metabase
# fix relative date time x-axis ticks

Closes https://github.com/metabase/metabase/issues/40935

### Description

Fixes relative date time x-axes may have duplicated ticks. When a relative date time unit `day-of-week` is selected, the x-axis values are numbers formatted as days of week. ECharts may pick intermediate values for ticks like `1.5` that will appear as duplicates with formatting applied. This PR sets minInterval to numeric x-axes which is 1 for relative date time columns.

### How to verify

- Create a chart summarized by a datetime dimension with day-of-week bucket
- Filter out all days except Monday and Tuesday
- Ensure there are no duplicated ticks


### Demo

Before
<img width="1702" alt="Screenshot 2024-04-03 at 10 31 44 PM" src="https://github.com/metabase/metabase/assets/14301985/7eff3e97-18a3-45dd-b9f6-bb523b19359a">

After
<img width="1698" alt="Screenshot 2024-04-03 at 10 24 00 PM" src="https://github.com/metabase/metabase/assets/14301985/df5ee46a-4ca5-47ff-b0b3-cc82d0624605">


### Checklist
